docs/source/build_network.rst

```diff
@@ -1,1466 +1,16 @@
 Building Your First Network
 ===========================
 
-.. note:: The Build your first network (BYFN) tutorial has been deprecated. If
+.. note:: The Build your first network (BYFN) tutorial has been removed. If
```

Comment:
You didn't do this, but can we fix the formatting on this, the block should be:

```
.. note:: The Build your first network (BYFN) tutorial has been deprecated.
   If you are getting started with Hyperledger Fabric and would like to deploy
   a basic network, see :doc:`test_network`. If you are deploying Fabric
   in production, see the guide for :doc:`deployment_guide_overview`.
```

Currently it looks like this with your change which does line up properly and the first line bolded with the `If` on it looks incorrect:

![image](https://user-images.githubusercontent.com/9400927/85892627-41df2100-b7bf-11ea-8e84-0c5e70a0dea5.png)

Reply:
The underscores also need to be double underscores `__` right now its not parsing the URL into the text

Reply:
Thanks, a little rusty in rst.

Reply:
I think we got it now.